### ISSUE TITLE
[BUG] TypeError: Cannot read properties of undefined (reading 'show')

### ISSUE THREAD

**Issue by spix18** [vlc-rpc/vlc-discord-rpc#10]
im getting this error with only this episode , other episodes are fine and even other movies, idk if its the episode itself but i thought i should report in case it is not.

ps: im gonna keep this episode in my pc in case you have an idea how to fix this so i can test it

````PS C:\Users\Spix\Desktop\vlc-discord-rpc-main> node .
Connecting to Discord...
Logged in as **************
VLC not playing; going to sleep. true
{
  _STATISTICS_WRITING_APP: "mkvmerge v82.0 ('I'm The President') 64-bit",
  showName: 'Shogun 2024',
  DURATION: '00:52:34.083000000',
  seasonNumber: '01',
  filename: 'Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA.mkv',
  BPS: '41',
  title: 'PSArips.com | Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA',
  _STATISTICS_TAGS: 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES',
  NUMBER_OF_FRAMES: '554',
  NUMBER_OF_BYTES: '16271',
  episodeNumber: '03',
  _STATISTICS_WRITING_DATE_UTC: '2024-03-05 06:51:36'
}
TypeError: Cannot read properties of undefined (reading 'show')
    at searchShow (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\Images\searchShow.js:8:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:42:18)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:13:25
C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:44
    if (show.image) {
             ^

TypeError: Cannot read properties of null (reading 'image')
    at module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:44:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:13:25

Node.js v20.11.1
PS C:\Users\Spix\Desktop\vlc-discord-rpc-main>

**Comment by Lauren1066** [comment]
Should be fixed in https://github.com/vlc-rpc/vlc-discord-rpc/pull/11 can you confirm?

**Comment by spix18** [comment]
> Should be fixed in #11 can you confirm?

yes the video does not exit anymore but there is still this error in the cmd (does not bother me) just letting you know .

````PS C:\Users\Spix\Desktop\vlc-discord-rpc-main> node .
Connecting to Discord...
Logged in as ***********
VLC not playing; going to sleep. true
{
  _STATISTICS_WRITING_APP: "mkvmerge v82.0 ('I'm The President') 64-bit",
  showName: 'Shogun 2024',
  DURATION: '00:52:34.083000000',
  seasonNumber: '01',
  filename: 'Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA.mkv',
  BPS: '41',
  title: 'PSArips.com | Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA',
  _STATISTICS_TAGS: 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES',
  NUMBER_OF_FRAMES: '554',
  NUMBER_OF_BYTES: '16271',
  episodeNumber: '03',
  _STATISTICS_WRITING_DATE_UTC: '2024-03-05 06:51:36'
}
TypeError: Cannot read properties of undefined (reading 'show')
    at searchShow (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\Images\searchShow.js:8:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:42:18)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:13:25
{
  _STATISTICS_WRITING_APP: "mkvmerge v82.0 ('I'm The President') 64-bit",
  showName: 'Shogun 2024',
  DURATION: '00:52:34.083000000',
  seasonNumber: '01',
  filename: 'Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA.mkv',
  BPS: '41',
  title: 'PSArips.com | Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA',
  _STATISTICS_TAGS: 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES',
  NUMBER_OF_FRAMES: '554',
  NUMBER_OF_BYTES: '16271',
  episodeNumber: '03',
  _STATISTICS_WRITING_DATE_UTC: '2024-03-05 06:51:36'
}
TypeError: Cannot read properties of undefined (reading 'show')
    at searchShow (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\Images\searchShow.js:8:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:42:18)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:13:25
{
  _STATISTICS_WRITING_APP: "mkvmerge v82.0 ('I'm The President') 64-bit",
  showName: 'Shogun 2024',
  DURATION: '00:52:34.083000000',
  seasonNumber: '01',
  filename: 'Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA.mkv',
  BPS: '41',
  title: 'PSArips.com | Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA',
  _STATISTICS_TAGS: 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES',
  NUMBER_OF_FRAMES: '554',
  NUMBER_OF_BYTES: '16271',
  episodeNumber: '03',
  _STATISTICS_WRITING_DATE_UTC: '2024-03-05 06:51:36'
}
TypeError: Cannot read properties of undefined (reading 'show')
    at searchShow (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\Images\searchShow.js:8:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:42:18)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:13:25
{
  _STATISTICS_WRITING_APP: "mkvmerge v82.0 ('I'm The President') 64-bit",
  showName: 'Shogun 2024',
  DURATION: '00:52:34.083000000',
  seasonNumber: '01',
  filename: 'Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA.mkv',
  BPS: '41',
  title: 'PSArips.com | Shogun.2024.S01E03.1080p.10bit.WEBRip.6CH.x265.HEVC-PSA',
  _STATISTICS_TAGS: 'BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES',
  NUMBER_OF_FRAMES: '554',
  NUMBER_OF_BYTES: '16271',
  episodeNumber: '03',
  _STATISTICS_WRITING_DATE_UTC: '2024-03-05 06:51:36'
}
TypeError: Cannot read properties of undefined (reading 'show')
    at searchShow (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\Images\searchShow.js:8:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async module.exports (C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\rpc-format.js:42:18)
    at async C:\Users\Spix\Desktop\vlc-discord-rpc-main\Functions\client.js:28:35

**Comment by Lauren1066** [comment]
Should be fixed in latest push.
